### PR TITLE
Finish top navigation bar

### PR DESCRIPTION
**Pull request by meghakak**

### CODE REVIEW

File: portfolio/src/main/webapp/blog.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="#ff9999">
+    <h1 class="title">STEP Blog</h1>
+    <div class="top-navigation"> 
+        <a href="index.html">Home</a>
+        <a href="hobbies.html">Hobbies</a>
+        <a href="gallery.html">Gallery</a>
+        <a href="blog.html">STEP Blog</a>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/blog.html]: nit: Indent by 2 spaces at a time. Don’t use tabs or mix tabs and spaces for indentation.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

I think these are indented by 4 spaces, please change to 2 to be consistent with rest of the file

File: portfolio/src/main/webapp/gallery.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="#ff9999">
+    <h1 class="title">Gallery</h1>
+    <div class="top-navigation"> 
+        <a href="index.html">Home</a>
+        <a href="hobbies.html">Hobbies</a>
+        <a href="gallery.html">Gallery</a>
+        <a href="blog.html">STEP Blog</a>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/gallery.html]: nit: Indent by 2 spaces at a time. Don’t use tabs or mix tabs and spaces for indentation.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

I think these are indented by 4 spaces, please change to 2 to be consistent with rest of the file

File: portfolio/src/main/webapp/hobbies.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="#ff9999">
+    <h1 class="title">My Hobbies</h1>
+    <div class="top-navigation"> 
+        <a href="index.html">Home</a>
+        <a href="hobbies.html">Hobbies</a>
+        <a href="gallery.html">Gallery</a>
+        <a href="blog.html">STEP Blog</a>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/hobbies.html]: nit: Indent by 2 spaces at a time. Don’t use tabs or mix tabs and spaces for indentation.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

I think these are indented by 4 spaces, please change to 2 to be consistent with rest of the file

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,14 +1,24 @@
...
   </head>
-  <body>
+
+  <body bgcolor="#ff9999">
+    <h1 class="title">Megha Kak</h1>
+    <div class="top-navigation"> 
+        <a href="index.html">Home</a>
+        <a href="hobbies.html">Hobbies</a>
+        <a href="gallery.html">Gallery</a>
+        <a href="blog.html">STEP Blog</a>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/index.html]: nit: Indent by 2 spaces at a time. Don’t use tabs or mix tabs and spaces for indentation.
https://google.github.io/styleguide/htmlcssguide.html#Indentation

I think these are indented by 4 spaces, please change to 2 to be consistent with rest of the file

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,15 +1,32 @@
+.title {
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/style.css:9]: nit: Please change back to the two space indent style for the file

https://google.github.io/styleguide/htmlcssguide.html#Indentation

File: portfolio/src/main/webapp/blog.html [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+<html>
+
+  <head>
+    <meta charset="UTF-8">
+    <title>STEP Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+
+  <body bgcolor="#ff9999">
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/blog.html]: Can this be moved to style.css? 

From checking validity, it seems like the bgcolor attribute is obsolete?
https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity

Same happens to the `<hr color=black>` tag as well.

And if so could you change this for the other files as well? Thanks!

File: portfolio/src/main/webapp/index.html [hunk 7]
```diff
@@ -1,14 +1,24 @@
...
+    </div>
+    <hr color=black>
+
     <div id="content">
-      <h1>Megha's Portfolio</h1>
       <p>Thank you for checking out my portfolio! My name is Megha and I'm a rising junior at UC Irvine majoring in 
         Computer Science and Engineering. I like to be involved on campus, to make my family and friends laugh, 
         and to put together music playlists for my friends!
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/index.html]: Sorry I missed this last time but could you add an alt attribute to the image below as well. And remove the `/` from the img end tag as it is not needed

https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback

**Review by linhnguye-g — CHANGES_REQUESTED** [review]
Sorry for all the nits, want to make sure style is consistent and clean.

It seems you are copying the same code for the nav bar into multiple files. Generally it's best to avoid having the exact same lines in multiple places. Imagine in a larger codebase, if you want to make a change to the nav bar, you now have to edit all the pages instead of a single place. This can get pretty complicated, especially if a file is missed.

I'm not entirely sure the best way to simplify this in HTML, it would be good to ask the Virtual Support Engineers about how you could reuse the nav bar on multiple pages.

File: portfolio/src/main/webapp/hobbies.html [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+    </div>
+    <hr color=black>
+
```
Comment (christophervo-g) [inline portfolio/src/main/webapp/hobbies.html]: nit: unnecessary blank line

**Review by christophervo-g — COMMENTED** [review]
Looks mostly good! One minor nit in addition to all the style nits Linh pointed out :)

File: portfolio/src/main/webapp/blog.html (hunk 6 above)
Comment (linhnguye-g) [reply portfolio/src/main/webapp/blog.html]: Also for completeness, just saw this: https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

File: portfolio/src/main/webapp/blog.html (hunk 1 above)
Comment (meghakak) [reply portfolio/src/main/webapp/blog.html]: Sorry about all the wrong indentations and thank you for catching them!

File: portfolio/src/main/webapp/blog.html [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>STEP Blog</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body class="background">
+    <h1 class="title">STEP Blog</h1>
+    <div id="top-navigation"> </div>
```
Comment (linhnguye-g) [inline portfolio/src/main/webapp/blog.html]: nit: Are the spaces between the opening and closing div needed here? If not please remove! Please do the same for the other files as well!

**Review by linhnguye-g — APPROVED** [review]
LGTM! Please do remove the space if it isn't necessary before submitting. Great job Megha!

**Review by christophervo-g — APPROVED** [review]
LGTM!

**Pull request merged**